Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -759,6 +760,33 @@ public override bool ContainsBlock(BlockHash blockHash)
             return false;
         }
 
+        /// <inheritdoc cref="BaseStore.PutTxIdBlockHashIndex(TxId, BlockHash)"/>
+        public override void PutTxIdBlockHashIndex(TxId txId, BlockHash blockHash)
+        {
+            _txExecutionDb.Put(
```

Comment:
Is it OK for you to share the same db with TxExecution? @dahlia 
IMHO, txExecutionDB consists of transient and deterministic data. So they can share the same db.  #1321

Reply:
cdf2e8d